kcards/app.py

```diff
@@ -27,6 +27,7 @@ def configure_logging(app):
 
 
 def register_blueprints(app):
+    app.register_blueprint(views.room.blueprint)
```

Comment:
Probably minimally by api and front facing maybe in `register_frontend()` and `register_backend()`?